app/src/Components/Event/EventDetails.tsx

```diff
@@ -56,6 +58,8 @@ interface EventDetailsProps {
  * @returns {React.ReactElement} Component
  */
 const EventDetails: FC<EventDetailsProps> = ({ event }) => {
+  console.log(Config)
```

Comment:
Please remove these two lines of console logging -- generally, console logging shouldn't be included in finished code unless it's logging errors, or there's another strong reason